src/app/my-messages/member-left-community-notification.ts

```diff
@@ -1,8 +1,10 @@
 import { AbstractNotification } from './abstract-notification';
 import { Community } from '../communities/community';
+import { User } from '../users/user';
 
 export class MemberLeftCommunityNotification extends AbstractNotification {
   community: Community;
+  user: User;
```

Comment:
Is it necessary to add this field? Are we going to remove this field in future as it does not make sense for the authenticated user.

Reply:
We talked about MemberKickedOutNotification 😉 
But MemberLeftCommunityNotification must contain a user to show manager which user has left a community.

Reply:
Yes, you are right.